Gemfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# gem "rails"
+
+gem 'rubocop', require: false
```

Comment:
This is the really good idea. It means you don't need to think about syntax things too much.